tests/test_clan_members_war.py

```diff
@@ -59,15 +59,15 @@ def test_clan_river_race_history(self):
         )
 
         # Testing if three players is returned based on get clan info
-        clan_players_war_history = clan_members_war.clan_river_race_history(clan_tag)
+        clan_info, clan_players_war_history = clan_members_war.clan_river_race_history(clan_tag)

```

Comment:
/supernit if you're not using that variable for anything you can just do this instead lol
```python
_, clan_players_war_history = clan_members_war.clan_river_race_history(clan_tag)
```

Or if you wanna check it returns the right thing, could do a 
```python
self.assertEqual(clan_info, clash_royale_client_responses.CLAN_INFO_API_RESPONSE)
```

Same for test you updated below

Reply:
done and updated the test document.